modular_skyrat/modules/customization/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -380,6 +380,8 @@
 /obj/item/proc/wear_species_version(file_to_use, state_to_use, layer, species, default_file_to_use)
 	return
 
+//generates a species specific icon for a state_to_use icon in file_to_use if one doesnt exist
+//said species specific icon will be generated using default_file_to_use
```

Comment:
It would be nice to indicate what each parameter does in your proc. I can show you a useful extension for VSC to make comment blocks easily, which really help with making good documentation.